test/Exploratory_Benchmarks/src/Runtime/Panics_And_Errors.enso

```diff
@@ -0,0 +1,88 @@
+from Standard.Base import all
```

Comment:
I added this test to `Exploratory_Benchmarks` - the suite that is _not_ run on CI by default, to avoid unnecessarily extending the runtime of the CI benchmark job. If we deem this is useful to have on CI, we can always move it.

Reply:
I need instructions to run these benchmarks via JMH, otherwise I am not able to perform deeper analysis of what is happening. What is the JMH command to invoke these benchmarks?

Reply:
I have moved to the main `Benchmarks` project and hidden the 'extra' variants behind an env variable.

If you want to enable the extra tests (that allow to compare performance across a few other scenarios), please define `ENSO_ENABLE_EXTRA_BENCHMARKS=1`.

Then one can run:
```
sbt "std-benchmarks/benchOnly .*Panics_And_Errors.*"
```

(I usually just run `std-benchmarks/benchOnly .*Panics_And_Errors.*` inside of already running the SBT 'REPL', but I assume adding the `"` should be enough to run it in batch mode.)

If I run it without extra env vars it will run the 2 tests: comparing Panics and Dataflow Errors:
```
(...)
[info] Benchmark                               Mode  Cnt    Score   Error  Units
[info] Panics_And_Errors_10000.Dataflow_Error  avgt    2  292,187          ms/op
[info] Panics_And_Errors_10000.Panic           avgt    2    0,246          ms/op
[info] Benchmark results reported into C:\NBO\enso\std-bits\benchmarks\.\bench-report.xml
[success] Total time: 87 s (01:27), completed 23 paâ”¼â•‘ 2023, 15:37:43
```